T/t8code/build_tarballs.jl

```diff
@@ -0,0 +1,112 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using Base.BinaryPlatforms
+const YGGDRASIL_DIR = "../.."
+include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
+
+name = "t8code"
+version = v"1.1.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/DLR-AMR/t8code/releases/download/v$(version)/t8code_v$(version).tar.gz",
+                  "0bd4bee6694735d14fb4274275fb8c4bdeacdbd29b257220c308be63e98be8f7"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd t8code/
+atomic_patch -p1 "${WORKSPACE}/srcdir/patches/mpi-constants.patch"
+
+# Set default preprocessor and linker flags
+# Note: This is *crucial* for Windows builds as otherwise the wrong libraries are picked up!
+export CPPFLAGS="-I${includedir}"
+export LDFLAGS="-L${libdir}"
+export CFLAGS="-O3"
+export CXXFLAGS="-O3"
+
+# Set necessary flags for FreeBSD
+if [[ "${target}" == *-freebsd* ]]; then
+  export LIBS="-lm"
+fi
+
+# Set necessary flags for Windows and non-Windodws systems
+FLAGS=()
+if [[ "${target}" == *-mingw* ]]; then
+  # Set linker flags only at build time (see https://docs.binarybuilder.org/v0.3/troubleshooting/#Windows)
+  FLAGS+=(LDFLAGS="$LDFLAGS -no-undefined")
+  # Configure does not find the correct Fortran compiler
+  export F77="f77"
```

Comment:
`F77`, really? :smiley:

Reply:
Right, this is not required anymore. Fixed in d3c1bad884f89a844f39c323d72fd621d9e5816a.